### PR TITLE
TextField - new outline preset

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
TextField - new outline preset

based on: https://github.com/wix/react-native-ui-lib/pull/3065

## Changelog
TextField - new outline preset

## Additional info
ticket 4172

### CODE REVIEW

File: demo/src/screens/componentScreens/TextFieldScreen.tsx [hunk 1]
```diff
@@ -74,6 +80,7 @@ export default class TextFieldScreen extends Component {
           />
           <TextField
             placeholder="Placeholder"
+            placeholderTextColor={Colors.$textNeutralLight}
```
Comment (ethanshar) [inline demo/src/screens/componentScreens/TextFieldScreen.tsx:83]: Why are you setting a different placeholderColor here?

File: demo/src/screens/componentScreens/TextFieldScreen.tsx [hunk 2]
```diff
@@ -261,33 +268,50 @@ export default class TextFieldScreen extends Component {
   }
 
   onChangeIndexFieldStyle = (index: number) => {
-    this.setState({customPreset: index === 0 ? 'underline' : 'outline'});
+    this.setState({preset: index === 0 ? 'underline' : 'outline'});
+  };
+
+  getDynamicFieldStyle = (context, props) => {
```
Comment (ethanshar) [inline demo/src/screens/componentScreens/TextFieldScreen.tsx:274]: This behavior can be achieved by passing a color object with all states, any reason you are doing it like this?

**Review by ethanshar — APPROVED** [review]
Looks good
Approved, see my comment on the example screen, it will help testing the component better

File: demo/src/screens/componentScreens/TextFieldScreen.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/TextFieldScreen.tsx:83]: I was just testing new token. Removed

File: demo/src/screens/componentScreens/TextFieldScreen.tsx (hunk 2 above)
Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/TextFieldScreen.tsx:274]: We don't have a prop to control the underline color with the ColorType... What are you referring to?

**Comment by Inbal-Tish** [comment]
@ethanshar I added another change to the demo's configuration file to allow overriding

**Pull request merged**